install/config/config.sh

```diff
@@ -29,6 +29,10 @@ sudo sed -i 's|^\(auth\s\+\[default=die\]\s\+pam_faillock.so\)\s\+authfail.*$|\1
 # Set Cloudflare as primary DNS (with Google as backup)
 sudo cp ~/.local/share/omarchy/default/systemd/resolved.conf /etc/systemd/
 
+# Install DNS configuration script to /usr/local/bin for system-wide access
+sudo cp ~/.local/share/omarchy/bin/omarchy-setup-cloudflare-dns /usr/local/bin/
+sudo chmod +x /usr/local/bin/omarchy-setup-cloudflare-dns
```

Comment:
We shouldn't need this. $OMARCHY_PATH/bin is now automatically part of path, so that should be enough that we can trigger it from anywhere.